bin/commander

```diff
@@ -8,30 +8,78 @@ program :version, Commander::VERSION
 program :description, 'Commander utility program.'
 
 command :init do |c|
-  c.syntax = 'commander init <file>'
+  c.syntax = 'commander init [option] <file>'
```

Comment:
Instead of `[option]` I would just put `[--modular]` since it's the only option.